settings/src/components/webauthn/list-keys.js

```diff
@@ -109,8 +112,8 @@ function ConfirmRemoveKey( { keyToRemove, onConfirm, onClose, deleting, error }
 				</div>
 			) : (
 				<div className="wporg-2fa__submit-actions">
-					<Button variant="primary" onClick={ onConfirm }>
-						Remove { keyToRemove.name }
+					<Button variant="primary" isDestructive onClick={ onConfirm }>
+						Delete
```

Comment:
IIRC, repeating the name was intentional, to make sure the user is aware _which_ key they're deleting. It'd be easy to click on a different key and then accidentally delete the wrong one. Similar to how github makes you type out the name of the repository when it confirms you want to delete it, but not that extreme.

I don't personally see the harm in repeating it, or the modal changing size. An alternative would be wrapping the title in a `<code>`, quotation marks, etc to distinguish it from the regular text.

I don't feel strongly, though.

Reply:
I understand the intention and think it can be effective. I'm not certain it's needed here, although it's probably too early to be completely certain of that. This action is undoable but isn't as significant as deleting a repository. It's more a kin to deleting an ssh key, and GitHub itself doesn't even reference the key in that window (although they use very strong language).

> I don't personally see the harm in repeating it
I don't know if its about harm, but more about readability. The repetition make me more likely to skip most of the content because I scan for the recurring pattern and skip past it. 
It can also look somewhat broken depending on what the the user named the key. 

I'll get a second opinion.

Reply:
IMO [this version](https://github.com/WordPress/wporg-two-factor/pull/237#discussion_r1323996058) is more pleasing.

I wonder whether bolding the name of the key might bring more attention to it, if we're concerned about ensuring the correct key.